vortex-datafusion/src/persistent/execution.rs

```diff
@@ -128,21 +132,57 @@ impl ExecutionPlan for VortexExec {
     fn repartitioned(
         &self,
         target_partitions: usize,
-        config: &ConfigOptions,
+        _config: &ConfigOptions,
```

Comment:
so we now ignore everything in this config?

Reply:
just when repartitioning, nothing very relevant right now as far as I can tell.